2_adventures/001/blndcat/04_day4/README.md

```diff
@@ -0,0 +1,34 @@
+# Day 4
+
+Rebase I
+
+Doing the following and understanding what is happening:
+edited .git/config and removed [remote "upstream"] lines
+then
+
+git remote add upstream https://github.com/RubySteps/21-day-challenge.git
+
+git fetch upstream
+
+git checkout master
+
+git rebase upstream/master
+pushed to github
+
+after that created day04 branch, dir, and this README
+
+* Notes
+Figured out the day ends/starts 3am local time. I think everyone should either calculate it or note the timestamp on the emails. 
+
+Next time I should try using a git command to remove the lines in .git/config
+
+Last time (day 2?) I created the new branch first, which lead to slightly different results.  I should probably take some time to work through and understand why this was.
+
+I will probably stick with RSpec for TDD for the time being
+
+During a presentation given by GitHub, I learnt that 
+git commit -m"commit message"
```

Comment:
A lot of Unix options will do this.  Some shells require no space.

As to the idea of -m"message" I avoid doing that, because the transition to an editor and using -v rather than -m means that I get to pause and really contemplate a good commit message and note to myself and others.

What does the -v do?  It pulls into the editor every thing that has changed for that commit, so that you can review and have a good chance to commit on the changes and catch anything that is out of place.

The information there doesn't end up staying in the commit message, of course.